schemes/utilities/qneg.F90

```diff
@@ -301,11 +306,18 @@ subroutine qneg_timestep_final(mpi_communicator, rootprocid, isrootproc,   &
       errcode = 0
       errmsg = ''
 
+      if (qneg_timestep_finalized) then
+         ! Only finalize the timestep once even if qneg_timestep_final is called multiple times
+         return
+      end if
+
```

Comment:
What should happen if qneg is finalized more than once?  Will the printed summary just be from the first place it is called?

Reply:
Yes, which should be OK because all of the qneg calculations that will be printed out in the final() call will have been done during the run phase.